Users/Install.rst

```diff
@@ -106,6 +106,13 @@ folder:
 
     $ virtualenv venv
 
+.. note::
+
+    If you have both Python3 and Python2 installed, use
+    ``virtualenv venv -p $(which python3)``.
+    This creates an isolated Python3 environment named ``venv``
+    in your current directory, as coala only works for Python >= 3.4 .
```

Comment:
1. I would rephrase and try to avoid mathematical notations:
   "... works for Python 3.4 and above."
2. No space between `3.4` and the full stop (automatically solved when you rephrase like suggested)